hnn_core/cell.py

```diff
@@ -640,3 +648,20 @@ def plot_morphology(self, ax=None, cell_types=None, show=True):
             The matplotlib 3D axis handle.
         """
         return plot_cell_morphology(self, ax=ax, show=show)
+
+    def update_end_pts(self):
```

Comment:
If you like my suggestion above, we could make this function private since it would no longer need to be user-facing.